src/foam/core/notification/services.jrl

```diff
@@ -288,3 +288,13 @@ p({
       .build();
   """
 })
+
+p({
+  class: "foam.core.boot.CSpec",
+  name: "notificationAgency",
+  lazy: false,
+  serve: false,
+  serviceScript: """
+    return new foam.lang.VirtualThreadAgency("notification-agency");
```

Comment:
Before we know how to monitor virtual thread, can we use normal thread pool instead?

Reply:
Now, we can monitor virtual threads via threads webagent.